### ISSUE TITLE
Empty data template?

### ISSUE THREAD

**Issue by eduardoarandah** [ratiw/vuetable-2-tutorial#103]
Is it possible to have a slot for empty data instead of "No Data Available" ?

![image](https://user-images.githubusercontent.com/4065733/44465068-14812500-a5e2-11e8-9b37-2fdb053583c9.png)

**Comment by a1tem** [comment]
Yes, I think this feature is really needed. I need to show only "Sorry, no data here!" text without showing columns.
